charts/otel-cloud-stack/values.yaml

```diff
@@ -737,3 +745,25 @@ logsCollector:
           exporters: [otlp]
 
 collectors: []
+
+opampBridge:
+  enabled: false
+  image: "ghcr.io/open-telemetry/opentelemetry-operator/operator-opamp-bridge:main"
+  endpoint: "wss://opamp.lightstep.com/v1/opamp"
+  headers:
+    "Authorization": "<lightstep api key>"
+  env:
+    - name: LIGHTSTEP_API_KEY
+      value: "[INSERT THIS VALUE]"
```

Comment:
I'm not exactly sure how to approach this, but try to follow how the Lightstep access token is rendered into the collector Helm charts (variable `${LS_TOKEN}`)